source/nordic_sdk/components/libraries/experimental_section_vars/section_vars.h

```diff
@@ -88,7 +88,7 @@
  *
  * @param[in]   section_name    Name of the section
  */
-#if defined __CC_ARM
+#if defined (_CC_ARM) || (__ARMCC_VERSION)
```

Comment:
The previous comment I meant about missing an underscore in `_CC_ARM` applies to most entries.

Reply:
👍 think I did a dodgy search and replace